### DISCUSSION TITLE
No puedo correr el test

### DISCUSSION THREAD

**Discussion by Tade07** [atralice/Curso.Prep.Henry#167] [General]
Estoy tratando de correr el test y no me salen las respuestas como correctas alguien me puede ayudar?
las probe en Javascrip y estan bien pero cuando quiero chequearlas con git bash me salen todas como failed.
Se agradece saludos!!

**Comment by bauticorrea** [comment]
Me pasa lo mismo, como lo solucionaste?

**Comment by Tade07** [comment]
Hola aun no pude solucionarlo y quedo ahí. No me respondio nadie aun!



Enviado desde mi Galaxy



-------- Mensaje original --------
De: bauticorrea ***@***.***>
Fecha: 14/3/2022 16:01 (GMT-03:00)
A: "atralice/Curso.Prep.Henry" ***@***.***>
CC: Tade07 ***@***.***>, Author ***@***.***>
Asunto: Re: [atralice/Curso.Prep.Henry] No puedo correr el test (Discussion #167)


Me pasa lo mismo, como lo solucionaste?

—
Reply to this email directly, view it on GitHub<https://github.com/atralice/Curso.Prep.Henry/discussions/167#discussioncomment-2359986>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWYHL4MOHHAB5DHJ4HLD6XDU76EHXANCNFSM5KE3UQQQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
